test/e2e/syn-grid.spec.coffee

```diff
@@ -0,0 +1,41 @@
+describe '<syn-grid />', ->
```

Comment:
Just testings clicks and key press. Rest of behaviours testing was made by unit tests